lib/formatter/parse-pattern.js

```diff
@@ -4,10 +4,11 @@ var ALPHA_REGEX = /[A-Za-z]/;
 var DIGIT_REGEX = /\d/;
 var WILD_REGEX = /./;
 var PLACEHOLDER_REGEX = /^[A-Za-z0-9\*]$/;
-var PLACEHOLDER_PATTERN_REGEX = '({{[^}]+}})';
+var PLACEHOLDER_PATTERN = '({{[^}]+}})';
 var PERMACHAR_REGEX = '(\\s|\\S)';
-var PATTERN_REGEX = new RegExp(PLACEHOLDER_PATTERN_REGEX + '|' + PERMACHAR_REGEX, 'g');
```

Comment:
It was creating a RegExp object versus creating a RegExp literal. @EvanHahn recommended that I remove the `new`.